STM32Cube/Core/Inc/FreeRTOSConfig.h

```diff
@@ -68,7 +68,7 @@
 #define configTICK_RATE_HZ                       ((TickType_t)1000)
 #define configMAX_PRIORITIES                     ( 56 )
 #define configMINIMAL_STACK_SIZE                 ((uint16_t)128)
-#define configTOTAL_HEAP_SIZE                    ((size_t)15360)
+#define configTOTAL_HEAP_SIZE                    ((size_t)150360)
```

Comment:
ignore this, was just done to make it work.
 the actual ioc change is done in another pr.